vipps-recurring-api.md

```diff
@@ -609,14 +623,15 @@ When charges are shown to users in Vipps, they will have a title, and a
 description. The title of a charge is derived directly from
 `{agreement.ProductName}` whereas the description is set per charge, i.e.
 `{charge.description}`. For example, a charge on an agreement with product
-name *Premier League subscription* with description *October* would look like
+name "Premier League subscription" with description "October" would look like
 the following screenshot:
 
 ![Charge description example](images/charge_descriptions_example.png)
 
-When the charge is processed, the payment will show up in the users's payment
-history. In the payment history a charge from Vipps recurring payment will have
-a description with follow format `{agreement.ProductName} - {charge.description}`.
+When the charge is completed (the money has been moved), the payment will
+show up in the users's payment history. In the payment history a charge from
+Vipps recurring payment will have a description with follow format
+`{agreement.ProductName} - {charge.description}`.
```

Comment:
This is a coming feature (jan 2022, most likely), the current solution shows the sale unit name for now

Reply:
Ok, but the test is already out there. I only changed the formatting.